agent/lib_aws_sdk_php.c

```diff
@@ -88,9 +88,9 @@ void nr_lib_aws_sdk_php_add_supportability_service_metric(
   }
 
   cp = buf;
-  strcpy(cp, PHP_AWS_SDK_SERVICE_NAME_METRIC_PREFIX);
+  nr_strcpy(cp, PHP_AWS_SDK_SERVICE_NAME_METRIC_PREFIX);
```

Comment:
for completeness, do you think [this strcpy](https://github.com/newrelic/newrelic-php-agent/pull/929/files#diff-3c0fd593a6a423b5ac09884a07389563329c49e50a9347f1968bfe43a02b0d3bR500) that also went into 11.0 also needs to be changed?

Reply:
Ah thanks - changed in commit [56808d1](https://github.com/newrelic/newrelic-php-agent/pull/949/commits/56808d1ca1ff6818a71f4ac84011fefd5d13f4f9).

Reply:
Please update PR title and description as it no longer matches the code change.